### ISSUE TITLE
Integration with other tools in a toolchain

### ISSUE THREAD

**Issue by gitvectors** [giuspen/cherrytree#2553]
When integrating CherryTree with other tools in a desktop orchestration I have suggested in earlier post that the absolute path to the current open node should be made available by a short cut which places path into clipboard.

Currently the only mechanism is to hover over the top toolbar and try to remember tha path before the popup clears.

I can get by with this popup but it does not fit naturally in an automated workflow across multiple documents scattered around.  I can locate *.ctd documents using Recoll by query ext:ctd (optionally adding a search pattern such as [ext:ctd "phrase"] and other multiple patterns available).

In short make it easier for handshakes between apps in a tool chain.

**Comment by gitvectors** [comment]
SOLVED

I use Recoll to list all *.ctd documents.
ext:ctd
Then I can right click on each *.ctd file to Copy URL and insert as links in any ctd file.
Thus we build cross links between *.ctd files.
